File: loopr-ios/Models/IndexResult.swift
//
//  IndexResult.swift
//  loopr-ios
//
//  Created by kenshin on 2018/12/24.
//  Copyright © 2018 Loopring. All rights reserved.
//

import Foundation

class IndexResult {
    var uuid: String
    var bullIndex: Int
    var bearIndex: Int
    var forwardNum: Int
    
    init(json: JSON) {
        self.uuid = json["uuid"].stringValue
        self.bullIndex = json["bullIndex"].intValue
        self.bearIndex = json["bearIndex"].intValue
        self.forwardNum = json["forwardNum"].intValue
    }
}

File: android/walletsdk/src/main/java/leaf/prod/walletsdk/model/response/crawler/IndexResult.java
/**
 * Created with IntelliJ IDEA.
 * User: kenshin wangchen@loopring.org
 * Time: 2018-11-23 2:38 PM
 * Cooperation: loopring.org 路印协议基金会
 */
package leaf.prod.walletsdk.model.response.crawler;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class IndexResult {

    private String uuid;

    private Integer bullIndex;

    private Integer bearIndex;

    private Integer forwardNum;
}
